molgenis-data-import/src/main/java/org/molgenis/data/importer/emx/EmxDataProvider.java

```diff
@@ -143,8 +142,7 @@ private Entity toEntity(EntityType entityType, Entity emxEntity)
 							entity.set(attrName, refEntities);
 						}
 						break;
-					case COMPOUND:
-						throw new RuntimeException(format("Illegal attribute type [%s]", attrType.toString()));
+					case COMPOUND://Since this is inside a "atomic attributes" loop this should not be possible
```

Comment:
Then it's IllegalStateException?

Reply:
This is now fixed, however this line has not changed so Github won't recognize that. IllegalState is thrown in the line below this one